lib/vehicle_factory.rb

```diff
@@ -0,0 +1,24 @@
+class VehicleFactory 
+    def initialize
```

Comment:
You don't need to even include this initialize when there's no arguments or attributes to set!